catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/ResourceCacheImplTest.java

```diff
@@ -409,17 +426,29 @@ public void containsSpecificResourceNotInCache() {
     }
 
     private Metacard createMetacard(String sourceId, String metacardId) throws URISyntaxException {
-        
+
         Metacard metacard = generateMetacard();
         metacard.setSourceId(sourceId);
         metacard.setAttribute(new AttributeImpl(Metacard.ID, metacardId));
         return metacard;
     }
 
     private ReliableResource createCachedResource(Metacard metacard) {
-        String fileName = "15bytes.txt";
+        String fileName = "10bytes.txt";
```

Comment:
It doesn't. See my general comment at the bottom. I had to change the file because after running the test, `15bytes.txt` disappeared on me. We can revert but I would have to re-verify.